httpserver-static.lua

```diff
@@ -3,16 +3,29 @@
 -- Author: Marcos Kirsch
 
 local function getMimeType(ext)
+   local gzip = false
    -- A few MIME types. Keep list short. If you need something that is missing, let's add it.
-   local mt = {css = "text/css", gif = "image/gif", html = "text/html", ico = "image/x-icon", jpeg = "image/jpeg", jpg = "image/jpeg", js = "application/javascript", json="application/json", png = "image/png"}
-   if mt[ext] then return mt[ext] else return "text/plain" end
+   local mt = {css = "text/css", gif = "image/gif", html = "text/html", ico = "image/x-icon", jpeg = "image/jpeg", jpg = "image/jpeg", js = "application/javascript", json = "application/json", png = "image/png"}
+   -- add comressed flag if file ends with gz
+   if ext:find("gz$") then 

```

Comment:
Seems like you are searching for "ends in gz".
It would be safer to search for "ends in .gz"